boards/arm/custom_nrf52840dk/board.cmake

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nrf52" "--speed=4000")
+board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
```

Comment:
Does the presence of `board_runner_args()` indicate you expect this repository to be exercised by `twister`?

Reply:
> Does the presence of `board_runner_args()` indicate you expect this repository to be exercised by `twister`?

This question seems to assume that these files exist for twister's sake. In fact their origin is in supporting the flash and debug commands which were added when west was first introduced. These options replaced makefile / environment variables which the old Kbuild system inherited from some shell scripts that were copied from the RIOT RTOS.